Jenkinsfile

```diff
@@ -97,6 +97,59 @@ pipeline {
             }
           }
         }
+        // We have 2 stages for GCE Authenticator tests. The first one runs inside
+        // a GCE instance and retrieves all the tokens that will be used in the tests.
+        // It then stashes the tokens, which are unstashed in the stage that runs the
+        // GCE Authenticator tests using the tokens.
+        // This way we can have a light-weight GCE instance that has no need for conjurops
+        // or git identities and is not open for SSH
+        stage('GCE Authenticator preparation - Allocate GCE Instance') {
+          steps {
+            script {
+              node('executor-v2-gcp-small') {
+                sh '''
+                  get_token_into_file() {
+                    local token_format="$1"
+                    local audience="$2"
+                    local filename="$3"
+
+                    curl \
+                      -s \
+                      -H 'Metadata-Flavor: Google' \
+                      "http://metadata/computeMetadata/v1/instance/service-accounts/default/identity?format=${token_format}&audience=${audience}" \
+                      > $filename
+                  }
+
+                  get_token_into_file "full" "conjur/cucumber/host/test-app" "gce_token_valid"
```

Comment:
The / in audience deos not require URL encoding?

Reply:
no